server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSReceiveOptionsCheckUtility.java

```diff
@@ -113,13 +117,17 @@ private void checkMap(CICSParser.Cics_receive_mapContext ctx) {
     }
     checkHasMutuallyExclusiveOptions("INTO or SET", ctx.INTO(), ctx.SET());
     checkHasMutuallyExclusiveOptions("TERMINAL or FROM", ctx.TERMINAL(), ctx.FROM());
+    if (noLengthOptionsEnabled() && (!ctx.INTO().isEmpty() || !ctx.FROM().isEmpty())) {
```

Comment:
This does not seem right. If I specify only `INTO` operand, the code both requires and bans the `LENGTH` keyword.
See above:
```
if (ctx.FROM().isEmpty()) checkHasIllegalOptions(ctx.LENGTH(), "LENGTH without FROM");
```